sonic_platform_base/chassis_base.py

```diff
@@ -333,6 +333,15 @@ def get_thermal(self, index):
 
         return thermal
 
+    def get_thermal_manager(self):
+        """
+        Retrieves thermal manager class on this chassis
+        :return: A class derived from ThermalManagerBase representing the
+        specified thermal manager. ThermalManagerBase is returned as default
+        """
+        from .sonic_thermal_control.thermal_manager_base import ThermalManagerBase
```

Comment:
Should not return ThermalManagerBase here. The implementation should in platform API.